runtime/src/accounts_db.rs

```diff
@@ -744,13 +744,25 @@ impl AccountsDB {
             dead_slots_w.len()
         };
         if dead_slots_len > 5000 {
-            self.process_dead_slots();
+            self.process_dead_slots(None);
         }
     }
 
-    pub fn process_dead_slots(&self) {
-        let empty = HashSet::new();
+    fn handle_reclaims_locked(&self, reclaims: SlotSlice<AccountInfo>) {
```

Comment:
nits: `handle_reclaims_completely`?

Reply:
Hmm, or this naming pair:

- `handle_reclaim` => `start_reclaimimg`
- `handle_reclaim_locked` => `ensure_reclaiming`

Or,

- `handle_reclaim` => `handle_reclaim`
- `handle_reclaim_locked` => `handle_and_wait_reclaim`/`handle_reclaim_and_wait`

Reply:
What I want to avoid is just adding the `_lock`, because it doesn't seem to add more information at least for me....

Reply:
@ryoqun yeah that's fair, how about `handle_reclaims_ensure_cleanup`

Reply:
Yeah, that got a lot better!